### DISCUSSION TITLE
Routing Help

### DISCUSSION THREAD

**Discussion by crotanite** [nuxt/nuxt#16945] [Questions]
The routes I'm trying to create are:
/news
/news/{category}
/news/{category}/{post}

e.g. /news/changelog/2022-09

But I would like /news and /news/{category} to be the same page.

I've tried creating the below directory structure:

news
  - [[category]]
    - [post]
      - index.vue
    - index.vue

The /news page works, but the /news/{category} pages route through to the [post] index.vue instead.

What is the correct way to organise the pages to handle this?

**Comment by Flanker72** [comment]
Seems it's better to place the [redirect](https://v3.nuxtjs.org/api/utils/define-nuxt-route-middleware#redirection) to `/news` from the `/news/{category}` to be more relevant.
You'll get a lot of duplicate pages if you'll render the same content for different categories if not to do so, it's very bad for SEO.

However you can create a component which will render the whole page and put it in both `index.vue` files to avoid code duplication.
